fsspec/tests/test_fuse.py

```
@@ -123,6 +123,7 @@ def test_chmod(mount_local):
         ["cp", str(mount_dir / "text"), str(mount_dir / "new")],
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
+        check=False,
```

Why the extra arg

---

https://docs.astral.sh/ruff/rules/subprocess-run-without-check/ Makes you think about whether you should enable to reraise an exception if the process fails (instead of having it fail silently). Happy to disable that if you prefer though.

---

I suppose `check=True` is reasonable, then. In a test like this, we would otherwise have the failure a couple of lines later. I'm not really sure why we're using subprocess at all in this case.

---

@martindurant Check=True made the test fail so I am just leaving it as is for now.